packages/sql-drizzle/src/pg.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @since 1.0.0
+ */
+import * as Client from "@effect/sql/Client"
+import { PgDeleteBase, PgInsertBase, PgSelectBase, PgUpdateBase } from "drizzle-orm/pg-core"
+import { PgRelationalQuery } from "drizzle-orm/pg-core/query-builders/query"
+import { PgRaw } from "drizzle-orm/pg-core/query-builders/raw"
+import { drizzle } from "drizzle-orm/postgres-js"
+import * as Context from "effect/Context"
+import * as Effect from "effect/Effect"
+import * as Layer from "effect/Layer"
+import { registerQueryBuilder } from "./internal/registry.js"
+import type { DrizzleDatabase } from "./internal/registry.types.js"
+
+export interface DrizzlePgDatabase extends
+  DrizzleDatabase,
+  Omit<
+    ReturnType<typeof drizzle>,
+    "run" | "all" | "get" | "values" | "transaction" | "execute" | "refreshMaterializedView" | "query" | "_"
+  >
+{}
+
+/**
+ * @since 1.0.0
+ * @category constructors
+ */
+export const make: Effect.Effect<DrizzlePgDatabase, never, Client.Client> = Effect.gen(function*() {
+  const client = yield* Client.Client
+  // instanciate the db without a client, since we are going to attach the client to the QueryBuilder
+  const db = drizzle({
+    options: {
+      parsers: [],
+      serializers: []
+    }
```

Comment:
This is to make drizzle patching of postgres-js client instance working. drizzle patching is not used in our case, since db instance is just used to convert query builder postgres dialect to sql